File: application/src/main/java/com/study/platform/domain/team/usecase/FindStudyTeamUseCase.java
package com.study.platform.domain.team.usecase;

import com.study.platform.domain.team.dto.response.StudyTeamResponse;

import java.util.UUID;

public interface FindStudyTeamUseCase {
    StudyTeamResponse execute(UUID teamId);
    StudyTeamResponse executeByPostId(UUID postId);
}

File: application/src/main/kotlin/com/study/platform/domain/team/usecase/FindStudyTeamUseCase.kt
package com.study.platform.domain.team.usecase

import com.study.platform.domain.team.dto.response.StudyTeamResponse
import java.util.UUID

interface FindStudyTeamUseCase {
    fun execute(teamId: UUID): StudyTeamResponse
    fun executeByPostId(postId: UUID): StudyTeamResponse
}
